### ISSUE TITLE
ComboBox has blank stop when isButtonAriaHidden=false (Narrator Scan Mode)

### ISSUE THREAD

**Issue by capucat** [microsoft/fluentui#21124]
### Environment Information

- **Package version(s)**: Fluent v7
- **Browser and OS versions**: Windows11

### Describe the issue:

We received an a11y bug regarding the dropdown chevron in our ComboBox:
""Actual Result:
Screen reader is not announcing name and role when focus is on 'Language dropdown' in scan mode.
It is announcing as 'Scan'.

Expected Result:
Screen reader should announce name and role when focus is on 'Language dropdown' in scan mode. ""

I have set isButtonAriaHidden=true and given the button an ariaLabel, but there's a blank stop on arrow down that announces 'scan'. It takes another arrow down to get narrator to read the label and role.

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/capucat/pen/RwLxLZj
#### Actual behavior:

When narrator is in scan mode, on arrow down there is a blank stop after the input. Another arrow down is necessary to make narrator announce the chevron button label and role.

#### Expected behavior:

On arrowDown, narrator should announce the next element's label and role (chevron button)

**Comment by gouttierre** [comment]
I’ve added this to our backlog for contributors as it’s low on our priority list and we’re likely not going to get to it for a while as we’re currently focused on larger coherence work items and enhancement
